analyzers/rolaguard_base_analyzer/BaseAnalyzerMain.py

```diff
@@ -80,15 +83,22 @@ def process_packet(packet, policy):
 
 
     chrono.start("checks")
-    # TODO: should a LAF-400 be emitted when a device reconnects? 
-    ## Check alert LAF-400
-    # # if device and not device.connected and policy.is_enabled("LAF-400"):
-    # #     emit_alert("LAF-400", packet, device=device, gateway=gateway,
-    # #                 number_of_devices = DataCollector.number_of_devices(packet.data_collector_id))
+    ## Check alert
+
+    ## LAF-404
+    # The data_collector and dev_eui is used as UID to count JRs.
+    jr_count = jr_counters[(packet.data_collector_id, packet.dev_eui)]
```

Comment:
In some cases dev_eui may be not set, will this cause any problem?

Reply:
You are right, fixed!